alignak/objects/contact.py

```diff
@@ -224,15 +224,19 @@ def get_groupname(self):
         :return: group name
         :rtype: str
         """
-        return self.contactgroups[0]
+        if self.contactgroups:
+            return self.contactgroups[0]
+        return 'Unknown'
```

Comment:
Yes for UI or macros ... make the user informed of a problem rather than hidding it.